app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnector.scala

```diff
@@ -35,6 +37,26 @@ class AgentAssuranceConnector @Inject()(@Named("agent-assurance-baseUrl") baseUr
     }
   }
 
+  def getActiveCesaRelationship(url: String)(implicit hc: HeaderCarrier): Future[Boolean] = {
```

Comment:
AgentAssuranceConnector needs monitoring, pls look at other connectors for details

Reply:
DesConnector got an example of monitoring upstream call